infra-gen2/backends/api/apiMultiAuth/amplify/auth/pre-sign-up-handler.ts

```diff
@@ -0,0 +1,4 @@
+import { PreSignUpTriggerHandler } from "aws-lambda";
```

Comment:
*nit:* `infra-gen2/backends/api/apiMultiAuth/amplify/auth/pre-sign-up-handler.ts` should be `infra-gen2/backends/api/api-multi-auth/amplify/auth/pre-sign-up-handler.ts` if following the naming conventions we generally follow for JS/TS projects.